inbound.go

```diff
@@ -415,6 +420,13 @@ func (response *InboundCallResponse) doneSending() {
 	if span := response.span; span != nil {
 		if response.applicationError || response.systemError {
 			ext.Error.Set(span, true)
+			errorType := appErrorType
+			if response.systemError {
+				errorType = systemErrorType
+				// if the error is a system error, set the error code in span log
+				span.LogKV("error-code", GetSystemErrorCode(response.err))
+			}
+			span.LogKV("error-type", errorType)
```

Comment:
What values does error type contain? Also, I suggest using the key `rpc.tchannel.app_error_type` instead, and using span tags if the values are well defined and bounded.

Reply:
Updated in the subsequent commit

Reply:
Changed to `rpc.tchannel.error_type`, since the error can be of type "application, or "system"